package-lock.json

```diff
@@ -20984,6 +20992,7 @@
           "resolved": "https://registry.npmjs.org/to-regex-range/-/to-regex-range-2.1.1.tgz",
           "integrity": "sha1-fIDBe53+vlmeJzZ+DU3VWQFB2zg=",
           "dev": true,
+          "optional": true,
           "requires": {
             "is-number": "^3.0.0",
             "repeat-string": "^1.6.1"
```

Comment:
Don't think this file needs to be included in these changes?

Reply:
@JamieB-gu this lock file manifested itself in the git push
All I did was npm i and perhaps the merge from main?